packages/grid/_modules_/grid/components/cell/GridSkeletonCell.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import clsx from 'clsx';
+import Skeleton from '@material-ui/lab/Skeleton';
```

Comment:
Tricky, this was moved in the core in v5. No idea how we could fix it. Maybe we can simply update #1902 once we get the changes in?

Reply:
Yes the simple checks for whether is it `v5` won't cut it.